core/chain-configs/src/updateable_config.rs

```diff
@@ -37,12 +37,12 @@ impl<T: Serialize> Serialize for MutableConfigValue<T> {
     }
 }
 
-impl<T: Copy + PartialEq + Debug> MutableConfigValue<T> {
+impl<T: Clone + PartialEq + Debug> MutableConfigValue<T> {
```

Comment:
Would it make sense to make the validator key Copy-able rather than this change?

Reply:
I tried but I found it better to relax the `Copy` requirement and use `Clone` instead. Reasons:
- it would require making Copy-able:
  - `AccountId` - which is considered stable and would require making a new release in separate repo. Perhaps there is a reason why it is non-Copy-able and it would not be possible to make it Copy-able anyway.
  - `PublicKey`, `SecretKey`, `ED25519PublicKey`, `ED25519SecretKey`
  - and possibly more
- `Copy` is for better perfomance but here we just reload simple structs rarely. 
- If we want to add non-Copy-able field in the future we would have to do this change anyway.